### DISCUSSION TITLE
Scoppy on RaspberryPi4B-2GB@PINN-LineageOS19(RPI4) with NikGapps

### DISCUSSION THREAD

**Discussion by GunoX9** [fhdm-dev/scoppy#94] [Show and tell]
I have successfully installed scoppy app and tested Raspberry Pico with USB2.0 port on RPI4.

Important note: Scoppy works sluggish on emulators. I DO NOT recommend using emulators on other linux distros.(Anbox, Waydroid) 

Don't even try on Windows.

**Comment by GunoX9** [comment]
Amateur shot by me:
![IMG_20230218_003832_428](https://user-images.githubusercontent.com/60858260/219798877-e8deac5a-214b-4a58-8b89-6e4a2e6b1de7.jpg)

If asked, I will add more.

**Reply by Askhento** [reply]
@GunoX9 please show some more info!
I have my rpi4 laying around and would like to have a basic oscilloscope for learning electronics.

**Reply by GunoX9** [reply]
Alright, do you know how to install android to rp?
https://electropeak.com/learn/how-to-install-android-on-raspberry-pi-tutorial/ there is a tutorial how to do it on this website but i don't like installing GAPPS after first boot.
Do your search and learn how to install Android with GAPPS. Installing GAPPS before first boot is healthy and prevents Google Apps to crash or become unresponsive.
After installing Android with GAPPS, just go to Play Store and download Scoppy app, and connect Raspbery Pico to RP4's USB. It just works.

**Reply by GunoX9** [reply]
Sorry for late response.
If a video will be more helpful to you, let me know. I will do a amateur recording for you.

**Reply by Askhento** [reply]
@GunoX9  Will definitely try!
I didn't know that android could be install in rpi))

**Reply by GunoX9** [reply]
@Askhento How is it going?

**Reply by Askhento** [reply]
> @Askhento How is it going?

ok I was succesfully installed android 
I've connected to raspberry via VNC, app working but  all I see is empty 

Ahhh I see i still need a pico board.... I thought for some reason that I will be able to use ADC from pri4 (((((

**Reply by GunoX9** [reply]
@Askhento yes, in my case, the Pi4 is a (for this situation) replacement for Android smartphone. I have LG G3 laying around but i find its 5 inchish screen small. Thus I wanted to use bigger screen.

I couldn't connect G3 to screen while using Pico and 5V Charger. I came up with this idea and tested it.

I shared this so people will be able to create a project where you can create multimeters, LCRs while using Cameras to stream etc.
In summary, you have a arm CPU that can be pushed with high loads and used as a hub. You can connect whatever you want it.
